nx-X11/lib/src/util/Makefile.am

```diff
@@ -0,0 +1,15 @@
+NULL =
+
+noinst_PROGRAMS=makekeys
+
+AM_CFLAGS =					\
+	$(X11_CFLAGS)				\
+	$(CWARNFLAGS)				\
+	$(NULL)
+
+AM_CPPFLAGS =					\
```

Comment:
> Add cross-compiling and lint sections commented out.

Nope. Seriously, I don't expect libNX_X11 to be split out of nxagent code base any time soon. In fact, the recommendation from Keith Packard here at DebConf today was actually move all libNX_X11 NX-specific code into nxagent directly and at the end drop libNX_X11 from our repo.